apps/test/unit/code-studio/pd/workshop_dashboard/attendance/session_attendance_test.jsx

```diff
@@ -98,6 +98,7 @@ describe('SessionAttendance', () => {
 
     // After the server responds
     server.respond();
+    wrapper.update();
```

Comment:
Throughout this PR, the issue was that [for `mount()`, updates are sometimes required when they weren't before.](https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#for-mount-updates-are-sometimes-required-when-they-werent-before)